sprinkler-timer-supplier/src/main/java/com/example/sprinkler/simulation_timer/TimerSimulationProperties.java

```diff
@@ -7,13 +7,26 @@
 
 @ConfigurationProperties(prefix = "simulation.timer")
 public class TimerSimulationProperties {
-	// first time produced
+
+	/**
```

Comment:
I'm not nitpicking but when I was doing the demo script I realized I did not understand the variables so I went through and explained them a bit and then added it to the code (javadocs)